Files/UserControls/SpecificPageOnStartup.xaml

```diff
@@ -0,0 +1,53 @@
+﻿<UserControl
+    x:Class="Files.UserControls.SpecificPageOnStartup"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Files.UserControls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    d:DesignWidth="400" Height="40">
+
+    <StackPanel Orientation="Horizontal" Margin="0,0,0,3">
+        <TextBox 
+            x:Name="PagePath"
+            Width="500"
+            Height="34"
+            Margin="12,0,0,0"
+            HorizontalAlignment="Left"
+            BorderThickness="1,1,0,1"
+            CornerRadius="2,0,0,2"
+            IsEnabled="False"
+            PlaceholderText="New Tab"
+            Text="" />
+        <Button
+            x:Name="btnMenu"
+            Height="34"
+            Content="&#xE712;"
+            CornerRadius="0,2,2,0"
+            FontFamily="Segoe MDL2 Assets"
+            Style="{ThemeResource AccentButtonStyle}"
+            ToolTipService.ToolTip="More options">
+            <Button.Flyout>
+                <MenuFlyout Placement="BottomEdgeAlignedRight">
+                    <MenuFlyoutItem
+                        x:Uid="ChangePage"
+                        Text="Change Page Location"
+                        Click="ChangePage_Click">
+                        <MenuFlyoutItem.Icon>
+                            <FontIcon Glyph="&#xE70F;" />
+                        </MenuFlyoutItem.Icon>
+                    </MenuFlyoutItem>
+                    <MenuFlyoutItem
+                        x:Uid="RemovePage"
+                        Text="Remove Page"
+                        Click="RemovePage_Click">
+                        <MenuFlyoutItem.Icon>
+                            <FontIcon Glyph="&#xE74D;" />
+                        </MenuFlyoutItem.Icon>
```

Comment:
fluent icons should be used

Reply:
How to use fluent icons? I used Segoe MDL2 icons from https://docs.microsoft.com/en-us/windows/uwp/design/style/segoe-ui-symbol-font, and I think that's part of fluent design?

Reply:
all icons have been changed and mdl2 icons are no longer used. See: #1374 


just change them to this:


![Anotación 2020-07-14 211257](https://user-images.githubusercontent.com/61259627/87491699-d39bba80-c616-11ea-8803-d5d0eb800f9f.png)

Reply:
Thank you for providing this. Where can I find a list of fluent icons on Microsoft's website?

Reply:
here: https://github.com/microsoft/fluentui-system-icons/blob/master/icons.md